index.html

```diff
@@ -491,64 +485,80 @@ <h2>
     </h2>
     <p>
       The <dfn data-local-lt="coordinator">credential request coordinator</dfn>
-      is a user-agent-defined component responsible for mediating [=digital
-      credential=] interactions within a given [=top-level traversable=]. It
-      ensures that only one interaction is active at any time across [=child
-      navigables=], and it manages the state transitions between different
-      [=credential request coordinator/interaction states=].
-    </p>
-    <p>
-      Each [=top-level traversable=] has a single associated [=credential
-      request coordinator=].
+      is a user-agent-defined component that mediates [=digital credential=]
+      interactions within a [=top-level traversable=]. Each [=top-level
+      traversable=] has exactly one associated coordinator. The coordinator
+      ensures that at most one interaction is active across all [=child
+      navigables=], orchestrates the end-to-end presentation or issuance flow,
+      and manages transitions between [=credential request
+      coordinator/interaction states=]. A user agent may delegate some or all
+      coordinator responsibilities to external wallet applications, platform
+      components, or other trusted entities according to user or platform
+      policy.
     </p>
     <p>
-      The coordinator manages the lifecycle of the current interaction's
-      {{Promise}}, including its resolution or rejection and integrates with an
-      {{AbortSignal}} to allow for both user-initiated and programmatic
-      cancellation.
+      The coordinator manages the lifecycle of the interaction's {{Promise}}
+      and its associated {{AbortSignal}}, including resolution with the user's
+      selected [=digital credential=] (or credentials) or rejection due to
+      errors or user/programmatic cancellation.
```

Comment:
selected [=digital credential|digital credential(s)=] or rejection due to
      errors or cancellation by user or program.